DotNet/emitter.js

```diff
@@ -102,6 +103,7 @@
 		addHook('deps', `using OpenQA.Selenium.Remote;`)
 		addHook('deps', `using System.Collections.Generic;`)
 		addHook('deps', `using System;`)
+		addHook('deps', `using System.Linq;`)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
		addHook("deps", `using System.Linq;`)
```